packages/plugin/webpack/test/WebpackConfig_spec.ts

```diff
@@ -172,7 +172,7 @@ describe('WebpackConfigGenerator', () => {
         },
       } as WebpackPluginConfig;
       const generator = new WebpackConfigGenerator(config, mockProjectDir, false, 3000);
-      const webpackConfig = await generator.getMainConfig();
+      const webpackConfig = generator.getMainConfig();
```

Comment:
I noticed in a handful of places we were using `await` for `getMainConfig` (which is synchronous)